api-controllers/ServiceFabrikApiController.js

```diff
@@ -347,7 +347,7 @@ class ServiceFabrikApiController extends FabrikBaseController {
           restore_guid: metadata.restore_guid,
           instance_guid: req.params.instance_id,
           arguments: _.assign({
-              backup: _.pick(metadata, 'type', 'secret')
+              backup: _.pick(metadata, 'type', 'secret', 'snapshotId', 'started_at', 'finished_at')
```

Comment:
Note: These extra parameters are needed for postgreSQL restore and distinguishing between PiTR and base-backup restore